util/sampling.go

```diff
@@ -99,7 +99,7 @@ func (rs *WeightedReservoirSample) Consider(value interface{}) {
 // computing x**1/weight may be ill-behaved.
 func (rs *WeightedReservoirSample) ConsiderWeighted(value interface{}, weight float64) {
 	if weight <= 0 {
-		glog.Warningf("reservoir sample received non-positive weight %f", weight)
+		clog.Warningf("reservoir sample received non-positive weight %f", weight)
```

Comment:
yes. I'll fix it. And by "fix" I mean I will delete the file since we're not using it anyways.